src/cargo/ops/cargo_update.rs

```diff
@@ -787,20 +800,23 @@ struct PackageChange {
     package_id: PackageId,
     previous_id: Option<PackageId>,
     kind: PackageChangeKind,
+    is_member: Option<bool>,
```

Comment:
I can't see the necessity of wrapping with an `Option` here except for representing “not relevant because it is a removed package”. Is that the reason of using it?

Reply:
Yes, I wanted to make sure that when I accessed it, this state was clear.  At first, I thought I was also going to have to act on it but that didn't end up happening.